src/clj/game/core/runs.clj

```diff
@@ -221,7 +221,7 @@
         trash-ab-cards (when (not= (:zone c) [:discard])
                          (->> (concat (all-active state :runner)
                                       (get-in @state [:runner :play-area]))
-                              (filter #(can-trigger? state :runner (:trash-ability (:interactions (card-def %))) % [c]))))
+                              (filter #(can-trigger? state :runner (->> % card-def :interactions :trash-ability) % [c]))))
```

Comment:
This double arrow `->>` could be a single arrow `->` to save a valuable character

Reply:
Honestly, I just forgot that the single threading macro places it immediately after the func. I keep thinking it places it to the left of the function, lol.